src/hooks/ProcessedRichNote/ProcessedRichNote.ts

```diff
@@ -1,12 +1,19 @@
 import { useEffect, useMemo, useState } from 'react'
+import { formatIpfsLink } from 'utils/ipfs'
 
 import { ProjectPreferences } from 'constants/v2/projectPreferences'
-import { loadAllMediaLinksPerLine } from './loadAllMediaLinksPerLine'
+import { loadAllMediaLinks } from './loadAllMediaLinks'
 
+// Gets strings that start with 'https'
 const URLRegex = new RegExp(
   /((?:https?):\/\/(?:\w+:?\w*)?(?:\S+)(:\d+)?(?:\/|\/([\w#!:.?+=&%!\-/]))?)/gi,
 )
 
+// Gets strings that start with 'ipfs'
+const ipfsLinkRegex = new RegExp(
```

Comment:
I reckon just combine the regex and use one matcher wdyt

Reply:
I separated them out to know when to call the `formatIpfsLink` function which converts the `ipfs://` format to `https://`

Reply:
See line 38